test/cpp/test_aten_xla_tensor.cpp

```diff
@@ -5173,7 +5173,12 @@ TEST_F(AtenXlaTensorTest, TestOneIndexTransfer) {
   }
 }
 
-// Temporarily disable test. See  https://github.com/pytorch/xla/issues/4501
+// Temporarily disable test. Original issue
```

Comment:
I guess this comment is not relevant to the master?

Reply:
This test is also disabled in the master branch https://github.com/pytorch/xla/blob/889a10390e388e29d804e23c99b566238c884740/test/cpp/test_aten_xla_tensor.cpp#L5176.

The comment also applies as both branches are impacted by the `nonzero` regression.